src/main/java/com/example/backendspringboottechiteasycontrollerles10/controllers/TelevisionControllerBonus.java

```diff
@@ -0,0 +1,56 @@
+package com.example.backendspringboottechiteasycontrollerles10.controllers;
+
+import com.example.backendspringboottechiteasycontrollerles10.exceptions.TelevisionNameTooLongException;
+import com.example.backendspringboottechiteasycontrollerles10.models.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+public class TelevisionControllerBonus {
+
+    public List<String> televisionDatabase = new ArrayList<>();
+
+
+    @GetMapping("/televisions")
+    public ResponseEntity<List<String>> getAllTelevisions() {
+      return ResponseEntity.ok(televisionDatabase);
+    }
+
+    @GetMapping("/televisions/{id}")
+    public ResponseEntity<Object> getTelevision(@PathVariable("id") int id) {
+        if (id >= 0 && id < televisionDatabase.size()) {
+            String t = televisionDatabase.get(id);
+            return ResponseEntity.ok(t);
+        } else {
+            return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
+        }
+    }
+
+    @PostMapping("/televisions")
+    // waarom krijg ik een foutmelding bij null?
```

Comment:
Welke null bedoel je?
Bedoel je null als input in Postman, of bedoel je de null in de return value?